src/apply/index.njk

```diff
@@ -9,7 +9,7 @@ noApplyCta: true
   {{ super() }}
   <div class="hero standard-restricted-width">
     <p class="hero__tag">You belong here</p>
-    <h1 class="hero__title g-max-4">Apply to be a founder at Ventures for Shared Prosperity</h1>
+    <h1 class="hero__title g-max-4">Apply to be an entrepreneur at Ventures for Shared Prosperity</h1>
```

Comment:
@will0101 It's OK to leave this in since you have it, but in the future best to not introduce sprawl like this. This page isn't one of the pages listed in #25. Best to keep PRs focused when possible

Reply:
Got it!